core/src/main/resources/hudson/model/DirectoryBrowserSupport/dir.jelly

```diff
@@ -50,13 +50,13 @@ THE SOFTWARE.
             ${%No files in directory}
            <j:if test="${showSymlinkWarning}">
              <p>
-                <img id="symlinkalert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" />
+                <img id="symlinkalert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" alt="symlink alert"/>
```

Comment:
Since the text in the next line exists, there is no need for an alt text here. This icon exists only to make the notice stand out.

Reply:
Resolved in8d861dd926cf9ee32b77b592a84b6c6bf143ef67